src/Async/index.js

```diff
@@ -322,6 +322,35 @@ function Async(fn) {
     }
   }
 
+  function bichain(l, r) {
+    if(!isFunction(l) || !isFunction(r)) {
+      throw new TypeError('Async.bichain: Functions required for both arguments')
```

Comment:
wanna use the `must` language as suggested by @bennypowers for this as well.
Also, we should make it clear that the functions need to be `Async` returning functions to help with debugging.